### PR TITLE
Fixed spelling error in translation.rst

### PR DESCRIPTION
**Pull request by premik91**
3 word spelling mistakes(typos) and 2 sentences restructured

### CODE REVIEW

**Comment by premik91** [comment]
(nothing serious, just did it because I was reading it and noticed)

File: docs/translation.rst [hunk 1]
```diff
@@ -4,17 +4,16 @@ Translation of PiplMesh
...
    
 Creation of translation files
 -----------------------------
-   
+
 .. warning:: It is important to run folowing commands from site root directory!
 
-Add your wanted language in ``settings.py`` under ``LANGUAGES``, make sure that in your Django applications
-you want translated there is a directory called ``locale``, if not you should make one.
+Add your wanted language(s) in ``settings.py`` under ``LANGUAGES`` and make sure your Django applications you want translated have a directory called ``locale``(if not you should create one).
```
Comment (mitar) [inline docs/translation.rst]: Comma after _if not_, before _you should create one_.

File: docs/translation.rst [hunk 2]
```diff
@@ -4,17 +4,16 @@ Translation of PiplMesh
 Making strings translatable
```
Comment (mitar) [inline docs/translation.rst:4]: Please wrap long lines.

File: docs/translation.rst [hunk 3]
```diff
@@ -4,17 +4,16 @@ Translation of PiplMesh
...
 ---------------------------
 
-All strings which do you want to be translated must be marked translatabe,
-in Django template like: ``{% trans "Some string" %}``,
-in Python files like: ``_("some string")``.
+To make your strings translatable edit them as follows
```
Comment (mitar) [inline docs/translation.rst]: Make this as a list, colon at the end and then a bullet list, no commas or dots at the end of each line.

**Comment by premik91** [comment]
Fixed :)

**Comment by mitar** [comment]
Please update from main repository.

**Comment by premik91** [comment]
Github for mac crashed and something was lost, so I had to remove previous branches(including this one). If the new branch isn't okay, I will recreate this one.

**Pull request closed without merging**